meshroom/ui/qml/main.qml

```diff
@@ -124,6 +140,22 @@ ApplicationWindow {
         onRejected: closed(Platform.Dialog.Rejected)
     }
 
+    MessageDialog {
+        id: unsavedComputeDialog
+
+        canCopy: false
+        icon.text: MaterialIcons.warning
+        preset: "Warning"
+        title: "Unsaved Project"
+        text: "Data will be computed in the default cache folder if project remains unsaved."
```

Comment:
Maybe add something like:
"Data will be computed in the default cache folder if project remains unsaved. If you compute and save your project later, the link to your data will be lost."